src/components/Home/Projects.tsx

```diff
@@ -30,255 +27,102 @@ export default function Projects() {
       link: "https://across.to",
     },
     {
       name: "Polymarket",
-      Icon: PolymarketIcon,
+      Icon: PolymarketLogo,
       link: "https://polymarket.com",
     },
   ];
 
   const middleRow = [
     {
       name: "Outcome",
-      Icon: OutcomeIcon,
+      Icon: OutcomeLogo,
       link: "https://outcome.finance",
     },
     {
       name: "Rated",
-      Icon: RatedIcon,
+      Icon: RatedLogo,
       link: "https://www.rated.network",
     },
     {
       name: "Shapeshift",
-      Icon: ShapeshiftIcon,
+      Icon: ShapeshiftLogo,
       link: "https://shapeshift.com",
     },
   ];
 
   const bottomRow = [
     {
       name: "Cozy",
-      Icon: CozyIcon,
+      Icon: CozyLogo,
       link: "https://www.cozy.finance",
     },
     {
       name: "Snapshot",
-      Icon: SnapshotIcon,
+      Icon: SnapshotLogo,
       link: "https://snapshot.org",
     },
     {
       name: "Sherlock",
-      Icon: SherlockIcon,
+      Icon: SherlockLogo,
       link: "https://www.sherlock.xyz",
     },
   ];
 
   return (
-    <OuterWrapper id={id} ref={ref}>
-      <InnerWrapper>
-        <TextWrapper>
-          <Header>Projects built with the OO</Header>
-          <DividerWrapper>
+    <section className="bg-white px-[--page-padding] pb-[128px] pt-[--header-blur-height]" id={id} ref={ref}>
+      <div className="mx-auto grid max-w-[--page-width] grid-cols-[1fr] grid-rows-[auto,auto] gap-12 lg:grid-cols-[1fr,1fr] lg:grid-rows-[1fr]">
+        <div className="row-start-1 lg:col-start-2 lg:max-w-[366px] lg:justify-self-end">
+          <h2 className="mb-3 text-4xl md:text-6xl lg:mb-6">Projects built with the OO</h2>
+          <div className="mb-6">
             <Divider />
-          </DividerWrapper>
+          </div>
           <AnimatedLink href="https://projects.uma.xyz/">All projects</AnimatedLink>
-        </TextWrapper>
-        <ProjectsWrapper>
+        </div>
+        <div className="row-start-2 grid grid-cols-[1fr] grid-rows-[auto,auto,auto] border-[0.5px] border-grey-400 lg:col-start-1 lg:row-start-1">
           <ProjectRow projects={topRow} line="top" />
           <ProjectRow projects={middleRow} line="middle" />
           <ProjectRow projects={bottomRow} line="bottom" />
-        </ProjectsWrapper>
-      </InnerWrapper>
-    </OuterWrapper>
+        </div>
+      </div>
+    </section>
   );
 }
 
 function ProjectRow({ projects, line }: { projects: Project[]; line: "top" | "middle" | "bottom" }) {
   return (
-    <ProjectRowWrapper
+    <div
+      className="grid grid-cols-[repeat(var(--num-columns),1fr)] grid-rows-[1fr]"
       style={
         {
           "--num-columns": line === "top" ? 2 : 3,
         } as CSSProperties
       }
     >
       {projects.map(({ name, link, Icon }) => (
-        <ProjectWrapper key={name} href={link} target="_blank">
-          <ArrowLink>
-            <ArrowIcon />
-          </ArrowLink>
-          <ProjectInnerWrapper>
-            <IconWrapper>
+        <NextLink
```

Comment:
the icons for the snapshot and rated projects were slightly too small and also would get cropped at some screen sizes. this is fixed now